kramer/src/main/kotlin/kramer/Aars.kt

```diff
@@ -0,0 +1,52 @@
+package kramer
```

Comment:
q: Do we need the license?

Reply:
Legally optional, since copyright is attached at creation, not based on whether you add license, and the whole project is licensed - but it ought to go in as a courtesy. I'll add license statements.